Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Abdellatif El Khlifi <abdellatif.elkhlifi@arm.com>

From 871521cf90dabb1634af4e47cdf198f979f1907d Mon Sep 17 00:00:00 2001
From: Abdellatif El Khlifi <abdellatif.elkhlifi@arm.com>
Date: Mon, 6 Dec 2021 15:53:56 +0000
Subject: [PATCH] corstone1000: dts: remove the use of fdt_addr_r

The device tree is embedded in the u-boot binary
and located at the end of the DDR. Its address
is specified in fdtcontroladdr environment variable.

No need to use fdt_addr_r anymore.

Signed-off-by: Abdellatif El Khlifi <abdellatif.elkhlifi@arm.com>
---
 include/configs/corstone1000.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/configs/corstone1000.h b/include/configs/corstone1000.h
index 7c6f66c891..31fc0cb29e 100644
--- a/include/configs/corstone1000.h
+++ b/include/configs/corstone1000.h
@@ -96,7 +96,6 @@
 #define CONFIG_EXTRA_ENV_SETTINGS				\
 				BOOTENV                         \
 				"usb_pgood_delay=250\0"         \
-				"fdt_addr_r=0x82000000\0"		\
 				"boot_bank_flag=0x08002000\0"		\
 				"kernel_addr_bank_0=0x083EE000\0"			\
 				"kernel_addr_bank_1=0x0936E000\0"			\
-- 
2.17.1

